src/course-unit/context/hooks.tsx

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+
+import { IframeContext, IframeContextType } from './iFrameContext';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
After update current branch to master branch and reinstall dependencies, still catch the error `ESLint: Prefer default export on a file with single export.(import/ prefer-default-export)`

Reply:
@bradenmacdonald, FYI.  Not sure what went wrong.

Reply:
Let's update `@openedx/frontend-build` in the next PRs to avoid delaying this feature, as it will require additional time to refactor the current code (I found 104 matches for `import/prefer-default-export`).

Reply:
I've done that refactor in https://github.com/openedx/frontend-app-authoring/pull/1519 and it's now merged to master :)